### PR TITLE
Warn users not to use dev-server in production

### PR DESCRIPTION
**Pull request by josh-berry**

### CODE REVIEW

File: temporalcli/commandsgen/commands.yml [hunk 1]
```diff
@@ -1400,8 +1400,17 @@ commands:
...
+
+      +------------------------------------------------------------------------+
+      | WARNING: The development server is not intended for use in production. |
+      | HTTP security checks have been relaxed to allow for easier local use.  |
+      |                                                                        |
+      | If you need a production-quality Temporal Service, please see:         |
+      | https://docs.temporal.io/production-deployment                         |
+      +------------------------------------------------------------------------+
+
+      View the Web UI for the default configuration at http://localhost:8233:
```
Comment (dandavison) [inline temporalcli/commandsgen/commands.yml:1413]: nit on pre-existing code: let's lose the terminal colon here. It's not helping anyone much and could well be an annoyance in a terminal emulator with a bad url-recognizing regex.

**Review by dandavison — APPROVED** [review]
LGTM! (one request)

**Review by yuandrew — APPROVED** [review]

**Comment by josh-berry** [comment]
Addressed @dandavison 's review comments and also addressed some comments from @fairlydurable offline.

**Comment by josh-berry** [comment]
Got a 👍 offline from @fairlydurable on the revised wording.

**Pull request merged**